src/fonts/README.md

```diff
@@ -1 +1,15 @@
 Icons are from [Icomoon](https://icomoon.io/app/). The selection in `selection.json` can be loaded and extended through the "Import icons" button.
+
+If you want to add icons, you can do that in icomoon by doing the following:
```

Comment:
<3 appreciate the notes

Reply:
Np! I might need to change these a bit due to some icomoon issues however.